conans/client/command.py

```diff
@@ -994,7 +994,7 @@ def remove(self, *args):
                             help="Remove only outdated from recipe packages. "
                                  "This flag can only be used with a reference")
         parser.add_argument('-p', '--packages', nargs="*", action=Extender,
-                            help="Select package to remove specifying the package ID")
+                            help="Remove all packages of the specified reference")
```

Comment:
This is good, but the previous one is still valid. ``--packages=xxxxx`` will remove that package with the xxxxx ID, and ``--packages`` will remove all packages.

Reply:
Okay then I think "Remove all packages of the specified reference if no specific package ID is provided"  describe the both ?

Reply:
Yes, good